src/transaction.js

```diff
@@ -625,24 +793,40 @@ Transaction.prototype.hashForZcashSignature = function (inIndex, prevOutScript,
     var hashSequence = this.getSequenceHash(hashType)
     var hashOutputs = this.getOutputsHash(hashType, inIndex)
     var hashJoinSplits = ZERO
+    var hashShieldedSpends = ZERO
+    var hashShieldedOutputs = ZERO
 
     var bufferWriter
-    var baseBufferSize = 4 * 5 + 32 * 4
+    var baseBufferSize = 0
```

Comment:
please comment what these operations below are doing
`4 * 5` why 5? What is 4 bits?
`32 * 4` why 4? What is 32 bits?